client/mysqlslap.c

```diff
@@ -1994,6 +2030,29 @@ pthread_handler_t run_task(void *p)
 
       if (con->limit && queries == con->limit)
         goto end;
+
+      if (unlikely(con->srv_time_target))
+      {
+        gettimeofday(&tmstamp, NULL);
+        /*
+          srv_time set as a ulong value based on the assumption that time diff
+          between two execs never exceed 4293sec (: ((2^32-1)-1000^2)/(1000^2) ).
+        */
+        ulong srv_time= (tmstamp.tv_sec - last_tmstamp.tv_sec) * 1000000 +
```

Comment:
I don't quite like srv_time and srv_time_target. I believe the names should be query_time and query_time_target. This goes better with the QPS naming of before.